### PR TITLE
feat: use alloy genesis now

### PR DESCRIPTION
**Pull request by supernovahs**

### CODE REVIEW

**Comment by mattsse** [comment]
I don't think we need this anymore

File: crates/primitives/src/genesis.rs [hunk 1]
```diff
@@ -1,1469 +1 @@
...
-/// // This will not return a signer, since the address is provided by the user and the signer
-/// // may be unknown.
-/// let addr = "0Ac1dF02185025F65202660F8167210A80dD5086".parse::<Address>().unwrap();
-/// allocator.add_funded_account_with_address(addr, U256::from(100_000_000_000_000_000u128));
-///
-/// // Once you're done adding accounts, you can build the alloc.
-/// let alloc = allocator.build();
-/// ```
-#[derive(Debug)]
-pub struct GenesisAllocator<'a> {
```
Comment (mattsse) [inline crates/primitives/src/genesis.rs:225]: we removed this type from alloy-genesis so we want to keep this here

**Review by mattsse — CHANGES_REQUESTED** [review]

**Comment by supernovahs** [comment]
1st issue pending, 2nd resolved

**Comment by gakonst** [comment]
to make it explicit, we want to remove the dependency on ethers genesis

**Review by mattsse — COMMENTED** [review]
thanks @supernovahs this is great!

I'll get rid of the compile errors, I think I know how to fix

**Review by mattsse — APPROVED** [review]

**Pull request merged**